<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketDialog</class>
 <widget class="QDialog" name="SocketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButtonTrame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Send trame</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QGroupBox" name="localAdapters">
     <property name="title">
      <string>Local Bluetooth Adapter</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="firstAdapter">
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="secondAdapter">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButton_Quit">
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Programme test d'envoie de trame</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="lineEditTrame"/>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QTextEdit" name="textEditRponse"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_connect_device_socket">
     <property name="text">
      <string>Selection périphérique socket</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
